src/policy/fees.cpp

```diff
@@ -54,6 +54,7 @@ bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_esti
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
+        {"EXPLICIT", FeeEstimateMode::EXPLICIT},
```

Comment:
The intent sounds more like no estimation so `{"NONE", FeeEstimateMode::NONE}`?

Reply:
I don't know. People could mistake that to mean "no fee".